pkg/urbit/worker/main.c

```diff
@@ -1035,6 +1036,7 @@ main(c3_i argc, c3_c* argv[])
   */
   {
     sscanf(wag_c, "%" SCNu32, &u3C.wag_w);
+    sscanf(hap_c, "%" SCNu32, &u3_Host.ops_u.hap_w);
```

Comment:
It'd be nice to have a warning here if the value is below some threshold (but i'm not sure what that threshold should be).